packages/relay/src/Environment.js

```diff
@@ -24,6 +25,7 @@ const GRAPHQL_URL = 'https://graphql.kiwi.com/';
 const store = new Store(new RecordSource());
 
 function fetchFromTheNetwork(networkHeaders, operation, variables, observer) {
+  Emitter.emit('load-started');
```

Comment:
Maybe something like `relay-load-started` would be better? Just in case you will use the emitter even for other things.